PackageExplorer/Utilities/StreamUtility.cs

```diff
@@ -9,5 +9,17 @@ public static Stream ToStream(string content)
         {
             return new MemoryStream(Encoding.UTF8.GetBytes(content));
         }
+
+        public static Stream MakeSeekable(Stream stream)
```

Comment:
The name comes from amazon 😄 
https://docs.aws.amazon.com/sdkfornet1/latest/apidocs/html/M_Amazon_S3_Util_AmazonS3Util_MakeStreamSeekable.htm